src/components/swimlanes/swimLanes.tsx

```diff
@@ -43,6 +43,7 @@ const SwimLanes = ({ lanesWithNumItems }) => {
               id={`card-${lane.slug}-${index}`}
               imageProps={{
                 alt: "",
+                isLazy: true,
```

Comment:
Does it _feel_ faster?...

Reply:
LOL

Reply:
Not that I'm noticing but I guess it's a matter of milliseconds anyways